Tests/selenium/CheckCreateUser.php

```diff
@@ -0,0 +1,147 @@
+<?php
+  /**
+   * Pre-requisites: phpunit, phpunit-selenium, selenium-standalone-server, chrome driver, php-curl extension
+   * 
+   * 
+   * @Matrix Israel Ltd.
+   */
+require_once __DIR__.'/../../vendor/autoload.php';
+
+class CheckCreateUserTest extends PHPUnit_Extensions_Selenium2TestCase
+{
+    const BROWSER = "chrome";
```

Comment:
Tried to change to firefox, but got an error and it looks like Selenium 2.53.0 and Firefox 47.0 are not compatible with each other(Ubuntu 16.04 seems to use Firefox 47.0).